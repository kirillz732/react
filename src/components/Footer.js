import React, {Component} from 'react';

import '../styles/style.scss';

export default class Footer extends Component {
  render() {
    return (
      <div className='footer'>
        ReactJS
      </div>
    )
  }
};
