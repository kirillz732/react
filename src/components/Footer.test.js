import React from 'react';
import { render } from '@testing-library/react';

import Footer from "./Footer";

describe('Footer', () => {
  test('render NoFilm snapshot', () => {
    const { asFragment } = render(<Footer/>);
    expect(asFragment()).toMatchSnapshot();
  })
});
