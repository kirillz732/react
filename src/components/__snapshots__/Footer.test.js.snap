// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer render NoFilm snapshot 1`] = `
<DocumentFragment>
  <div
    class="footer"
  >
    ReactJS
  </div>
</DocumentFragment>
`;
